airbnb_project/listings/README.md

```diff
@@ -1,64 +1,106 @@
-# Harvest Listings API
-
-This project includes a Django API endpoint that triggers a listings collections process using Scrapy. The view (
-`harvest_listings`) allows users to start the scraping process via a get request.
+# Airbnb Listings Harvester API
 
+A Django API service that harvests Airbnb listings data using Scrapy. 
 ## Prerequisites
 
-1. Have python3 installed
-2. Have all the poetry dependencies installed
-3. Have postgres server running
-4. Add the `AirBnBAPI_KEY` in the `.env` file
+- Python 3.x
+- Poetry for dependency management
+- Docker and Docker Compose (for database)
+
+## Setup
+
+### 1. Environment Configuration
 
-## Running the Server
+Create a `.env` file in the `airbnb_project` folder with the following variables:
 
-1. **Start the Django Development Server**:
+```env
+SECRET_KEY=your_secret_key
+DJANGO_DEBUG=True
+AIRBNB_PUBLIC_API_KEY=d306zoyjsyarp7ifhu67rjxn52tv0t20
```

Comment:
Can you please share a bit more context on this public key?

Reply:
The public API key is included in the API calls made by Airbnb's website. Without it, the API call that retrieves additional listing details after the ID has been scraped doesn't go through.

Reply:
So, this is a like a secret to us? We shouldn't be checking in this in code base. Thoughts?

Reply:
it is not really a secret to us. It is publicly available if you inspect requests made by the airbnb website. We can still keep it a secret.